lib/rtcpeerconnection.js

```diff
@@ -422,12 +429,22 @@ RTCPeerConnection.prototype.close = function close() {
 };
 
 /**
- * Construct an {@link RTCSessionDescription} containing an SDP answer.
- * @param {?function} [onSuccess]
- * @param {?function} [onFailure]
+ * onSuccess callback for {@link RTCPeerConnection.createAnswer}
+ * @callback onCreateAnswerSuccess
+ * @param {RTCSessionDescription}
+ *//**
+ * Construct an {@link RTCSessionDescription} containing an SDP offer.
+ * @param {onCreateAnswerSuccess} onSuccess
+ * @param {function} onFailure
+ *//**
+ * Construct an {@link RTCSessionDescription} containing an SDP offer.
  * @returns {Promise<RTCSessionDescription>}
  */
 RTCPeerConnection.prototype.createAnswer = function createAnswer(onSuccess, onFailure) {
+  if (!onSuccess) {
```

Comment:
I think we need to check if `onSuccess` is a function here—technically you could pass RTCAnswerOptions in the Promise-returning version (http://w3c.github.io/webrtc-pc/#interface-definition).